doc/odoc_for_authors.mld

```diff
@@ -375,14 +375,95 @@ OCaml code blocks can be written using the enclosing tags [ {[ ... ]} ].
 The code inside these blocks will be properly styled as OCaml in the generated
 doc.
 
-{v
+{delim@ocaml[
 (** This how one binds a variable in OCaml:
     {[
     let x = 0
     ]}
 *)
+]delim}
+
+Note that the content of the code block above is ["let x = 0"]. How is the initial content (["\n    let x = 0\n    "]) turned into that?
+
+{4:indentation_code_blocks Indentation in code blocks}
+
+Odoc strips whitespace from the content of a codeblock, so that it contains only what is in the following box:
+
+{v
+  {[
+  +-----------------------
+  |
+  |    content
+  |
+  +-----------------------
+  ]}
 v}
 
+In other words, [odoc]:
+{ol
+{- Removes the first line if it contains only whitespace,}
+{- Removes the last line if it contains only whitespace,}
+{- Deindents as much as the opening curly brace's column.}}
+
+Note that if there is non-whitespace content in the first new line, it is
+considered starting at the upper [+] in the box above:
```

Comment:
The `it` is a bit ambiguous. It's not clear if you talk about the line or the non-whitespace content.